src/rest/Endpoints.ts

```diff
@@ -269,4 +269,22 @@ export default class Endpoints {
   public static GUILD_VANITY_URL(guildID: string) {
     return `/guilds/${guildID}/vanity-url`;
   }
+
+  /**
+   * `/guilds/${guildID}/emojis/${emojiID}`
+   * - GET
+   * - PATCH
+   * - DELETE
+   */
+  public static EMOJI(guildID: string, emojiID: string) {
```

Comment:
This should probably named `GUILD_EMOJI`
```suggestion
  public static GUILD_EMOJI(guildID: string, emojiID: string) {
```

Reply:
Done